PredictOlympicMedals.ipynb

```diff
@@ -781,16 +773,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "'Average AUC score is 0.94 with standard dev 0.032'"
+       "'Average AUC score is 0.949 with standard dev 0.024'"
```

Comment:
In the Analysis section below, we can updat the std numbers to match

Reply:
Updated